packages/lib/sdk/src/plugins/datasources/updateManifest.spec.js

```diff
@@ -156,8 +161,31 @@ describe('updateManifest', () => {
 			locatedFiles: {
 				my_csv: ['x'],
 				needful_things: ['needful_things', 'orders']
+			},
+			locatedSchemas: ['my_csv', 'needful_things']
+		};
+		const dataDir = '/_evidence';
+		await fs.mkdir(path.join(dataDir), { recursive: true });
+		await fs.writeFile(
+			path.join(dataDir, 'manifest.json'),
+			JSON.stringify({ renderedFiles: fsManifest.renderedFiles })
+		);
+		await updateManifest(updatedManifest, dataDir);
+		expect(JSON.parse(await fs.readFile(path.join(dataDir, 'manifest.json'), 'utf8'))).toEqual({
+			renderedFiles: updatedManifest.renderedFiles
+		});
+	});
+	it('should remove an old source correctly', async () => {
+		const fsManifest = {
+			renderedFiles: {
+				needful_things: ['static/data/needful_things/orders/orders.parquet']
 			}
 		};
+		const updatedManifest = {
+			renderedFiles: {},
+			locatedFiles: {},
+			locatedSchemas: []
+		};
 		const dataDir = '/_evidence';
```

Comment:
@zachstence this is the test for that behavior

Reply:
Oh I missed it, nice